apps/reflect/src/server/router/twitchbot.ts

```diff
@@ -109,29 +109,35 @@ export const twitchBotRouter = t.router({
       const twitchAccId = twitchAccount?.providerAccountId;
       if (!twitchAccId) return;
 
-      const findConfig = await ctx.prisma.twitchBotConfigs.findFirst({
-        where: { configName: configName, twitchChannelId: twitchAccId },
-      });
-
-      if (findConfig) {
-        return await ctx.prisma.twitchBotConfigs.update({
+      configs.forEach(async config => {
+        const findConfig = await ctx.prisma.twitchBotConfigs.findFirst({
           where: {
-            id: findConfig.id,
+            configName: config.configName,
+            twitchChannelId: twitchAccId,
           },
+        });
+
+        if (findConfig) {
+          await ctx.prisma.twitchBotConfigs.update({

```

Comment:
I tried using Prisma upsert but either failed or something is missing in this scenario so Prisma upsert was invalid.